.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: go
+dist: trusty
+
+go:
+  - 1.8.1
+
+before_install:
+  - echo $PATH
+  - export PATH=`echo $PATH | sed -e 's/:\/usr\/local\/clang-3.5.0\/bin//'`
+  - wget http://security.ubuntu.com/ubuntu/pool/main/g/gcc-5/libstdc++6_5.4.0-6ubuntu1~16.04.4_amd64.deb
+  - mkdir ./libc
+  - dpkg-deb -x ./libstdc++6_5.4.0-6ubuntu1~16.04.4_amd64.deb ./libc
+  - export LD_LIBRARY_PATH=`pwd`/libc/usr/lib/x86_64-linux-gnu/
+  - wget http://releases.llvm.org/4.0.1/clang+llvm-4.0.1-x86_64-linux-gnu-debian8.tar.xz
+  - tar -xf ./clang+llvm-4.0.1-x86_64-linux-gnu-debian8.tar.xz
+  - sudo cp clang+llvm-4.0.1-x86_64-linux-gnu-debian8/bin/clang-format /usr/local/bin/
+  - which clang-format
+  - clang-format --version
+
+install: true
+
+script:
+  - go get ./...
```

Comment:
Otherwise it fails with:
```
$ make all all-tools
go install ./syz-manager ./syz-fuzzer
syz-manager/manager.go:21:2: cannot find package "github.com/google/syzkaller/dashboard/dashapi" in any of:
	/home/travis/gopath/src/github.com/xairy/syzkaller/vendor/github.com/google/syzkaller/dashboard/dashapi (vendor tree)
	/home/travis/.gimme/versions/go1.8.1.linux.amd64/src/github.com/google/syzkaller/dashboard/dashapi (from $GOROOT)
	/home/travis/gopath/src/github.com/google/syzkaller/dashboard/dashapi (from $GOPATH)
syz-manager/cover.go:18:2: cannot find package "github.com/google/syzkaller/pkg/cover" in any of:
	/home/travis/gopath/src/github.com/xairy/syzkaller/vendor/github.com/google/syzkaller/pkg/cover (vendor tree)
	/home/travis/.gimme/versions/go1.8.1.linux.amd64/src/github.com/google/syzkaller/pkg/cover (from $GOROOT)
	/home/travis/gopath/src/github.com/google/syzkaller/pkg/cover (from $GOPATH)
syz-manager/manager.go:23:2: cannot find package "github.com/google/syzkaller/pkg/csource" in any of:
	/home/travis/gopath/src/github.com/xairy/syzkaller/vendor/github.com/google/syzkaller/pkg/csource (vendor tree)
	/home/travis/.gimme/versions/go1.8.1.linux.amd64/src/github.com/google/syzkaller/pkg/csource (from $GOROOT)
	/home/travis/gopath/src/github.com/google/syzkaller/pkg/csource (from $GOPATH)
...
```

Do I need to do `go install ./...` instead?